web/src/app/main/components/ToolbarTitle.js

```diff
@@ -46,10 +46,10 @@ const mapSingular = {
 const nameQuery = typeName => gql`
   query($id: ID!) {
     data: ${typeName}(id: $id) {
-      id
-      name
+        id
```

Comment:
Spacing is still off in this. I thought prettier used to handle this automatically though. Anybody remember?

Reply:
It does not work for whatever reason. Cook couldn't get it to work yesterday either.